conn_test.go

```diff
@@ -47,7 +102,16 @@ func TestConn_DisconnectNotify(t *testing.T) {
 
 	t.Run("protocol error", func(t *testing.T) {
 		connA, connB := net.Pipe()
-		c := jsonrpc2.NewConn(context.Background(), jsonrpc2.NewPlainObjectStream(connB), nil)
+		c := jsonrpc2.NewConn(
+			context.Background(),
+			jsonrpc2.NewPlainObjectStream(connB),
+			noopHandler{},
+			// Suppress log message. This connection receives an invalid JSON
+			// message that causes an error to be written to the logger. We
+			// don't want this expected error to appear in os.Stderr though when
+			// running tests in verbose mode or when other tests fail.
+			jsonrpc2.SetLogger(log.New(io.Discard, "", 0)),
+		)
```

Comment:
This change doesn't actually have anything to do with the rest of this PR. I slipped this in because I kept seeing `jsonrpc2: protocol error: invalid character 'i' looking for beginning of value` in the test logs while working on tests. I introduced this test in 028a50bb (#64) and somehow missed that this test was spamming the test logs.

Reply:
I am only noticing now that this line is commented out. Wow, that's sloppy of me. Will fix.

Reply:
Fixed in f74a03be7a5abde73b015f396063044c210ff704.